### DISCUSSION TITLE
PM liquibase.lockservice - not working anymore since update - 1.2.2

### DISCUSSION THREAD

**Discussion by todwt** [nzymedefense/nzyme#657] [Questions] [answered]
Since the update 1.2.2 nzyme is not working anymore. I reinstalled it from the beginning and created it with a new config file. 

Details:
Raspberry PI 4
Nzyme  1.2.2
netstat: show no open ports
log: 
18:55:48.012 [main] INFO  horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader - Skipping disabled alert callback of type [email].
18:55:48.020 [main] INFO  horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader - Skipping disabled alert callback of type [file].

**Comment by lennartkoopmann** [comment]
What is the output of `journalctl -xe`?

**Comment by todwt** [comment]
Feb 14 16:06:34 defender nzyme[728]: Feb 14, 2022 4:06:34 PM liquibase.lockservice
Feb 14 16:06:34 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:06:38 defender kernel: cam-dummy-reg: disabling
Feb 14 16:06:38 defender kernel: cam1-reg: disabling
Feb 14 16:06:42 defender systemd[1]: systemd-fsckd.service: Succeeded.
░░ Subject: Unit succeeded
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░
░░ The unit systemd-fsckd.service has successfully entered the 'dead' state.
Feb 14 16:06:44 defender nzyme[728]: Feb 14, 2022 4:06:44 PM liquibase.lockservice
Feb 14 16:06:44 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:06:50 defender sudo[826]:       pi : TTY=pts/0 ; PWD=/home/pi ; USER=root ; COMMAND=/usr/bin/systemctl status nzyme
Feb 14 16:06:50 defender sudo[826]: pam_unix(sudo:session): session opened for user root(uid=0) by pi(uid=1000)
Feb 14 16:06:50 defender systemd[1]: systemd-hostnamed.service: Succeeded.
░░ Subject: Unit succeeded
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░
░░ The unit systemd-hostnamed.service has successfully entered the 'dead' state.
Feb 14 16:06:52 defender sudo[826]: pam_unix(sudo:session): session closed for user root
Feb 14 16:06:54 defender nzyme[728]: Feb 14, 2022 4:06:54 PM liquibase.lockservice
Feb 14 16:06:54 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:07:04 defender nzyme[728]: Feb 14, 2022 4:07:04 PM liquibase.lockservice
Feb 14 16:07:04 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:07:14 defender nzyme[728]: Feb 14, 2022 4:07:14 PM liquibase.lockservice
Feb 14 16:07:14 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:07:24 defender nzyme[728]: Feb 14, 2022 4:07:24 PM liquibase.lockservice
Feb 14 16:07:24 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:07:34 defender nzyme[728]: Feb 14, 2022 4:07:34 PM liquibase.lockservice
Feb 14 16:07:34 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:07:44 defender nzyme[728]: Feb 14, 2022 4:07:44 PM liquibase.lockservice
Feb 14 16:07:44 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:07:54 defender nzyme[728]: Feb 14, 2022 4:07:54 PM liquibase.lockservice
Feb 14 16:07:54 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:08:04 defender nzyme[728]: Feb 14, 2022 4:08:04 PM liquibase.lockservice
Feb 14 16:08:04 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:08:14 defender nzyme[728]: Feb 14, 2022 4:08:14 PM liquibase.lockservice
Feb 14 16:08:14 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:08:24 defender nzyme[728]: Feb 14, 2022 4:08:24 PM liquibase.lockservice
Feb 14 16:08:24 defender nzyme[728]: INFO: Waiting for changelog lock....
Feb 14 16:08:34 defender nzyme[728]: Feb 14, 2022 4:08:34 PM liquibase.lockservice
Feb 14 16:08:34 defender nzyme[728]: INFO: Waiting for changelog lock....

**Comment by todwt** [comment, marked as answer]
I found the error, it was the lock in postgres.

Solution:
Connect to postgres -> choose database nzyme -> remove entry from "databasechangeloglock".
